scripts/gitlab-build.sh

```diff
@@ -48,6 +48,8 @@ set_env_win () {
   echo "@ signtool sign /f "\%"1 /p "\%"2 /tr http://timestamp.comodoca.com /du https://parity.io "\%"3" > sign.cmd
 }
 build () {
+  echo "Remove index"
+  rm -rf cargo/registry/index/*.
```

Comment:
I guess you meant, `.cargo/registry/index`? That's a terrible idea! Cargo will have to redownload it's crates index and cargo builds may interfere with each.

Reply:
Cargo cache is in `cargo/` #7921: https://github.com/paritytech/parity/pull/7921/files

The path "is correct", but I can't evaluate if this is the correct way to address this. Problem is that windows builds fail due to a build lock even though there is no other build running.

https://gitlab.parity.io/parity/parity/-/jobs/80511

Please advise :)

Reply:
@General-Beck Let's wrap this in the conditional `if [[ "windows" = $IDENT ]]` for now?
I guess the actual root cause is our cache key selection, but I would do that as a part of enhancing the #7988

Reply:
@5chdn I believe this is not a correct way of addressing the problem